trailing_spaces.py

```diff
@@ -437,8 +437,11 @@ def freeze_last_version(self, view):
         # For some reasons, the on_activated hook gets fired on a ghost document
         # from time to time.
         if file_name:
-            on_disk = codecs.open(file_name, "r", "utf-8").read().splitlines()
-
+            try:
+                on_disk = codecs.open(file_name, "r", "utf-8").read().splitlines()
+            except FileNotFoundError:
+                sublime.status_message('Unable to freeze last version; are you sure %s exists?' % file_name)
+                return
```

Comment:
Current code on master looks like this:
```py
        if file_name and isfile(file_name) and not view.is_scratch():
            on_disk = codecs.open(file_name, "r", "utf-8").read().splitlines()
```
so I think it covers most if not all cases already.